manifests/0000_09_cluster-osin-operator_05_deploy.yaml

```diff
@@ -71,6 +71,13 @@ spec:
         env:
         - name: IMAGE
           value: quay.io/openshift/origin-hypershift:v4.0
+        - name: OPERATOR_IMAGE
+          value: docker.io/openshift/origin-cluster-authentication-operator:v4.0  # I believe this is supposed to exist
```

Comment:
The environment variable value does not match the actual image few lines above:
```
- name: operator
  image: quay.io/openshift/origin-cluster-authentication-operator:latest
```

Reply:
If you were adding a new image reference, you would probably have to add it to `manifests/image-references`, too

Reply:
It is not meant to reference the same image.  Just following pattern I saw in kube API server operator.  We do not have that docker image whereas kibe API server does for it's operator, so I added this as a reminder.  It is inert as-is.